app/controllers/application.js

```diff
@@ -1,6 +1,10 @@
 import Ember from 'ember';
 import OSFAgnosticAuthControllerMixin from 'ember-osf/mixins/osf-agnostic-auth-controller';
 
+import config from '../config/environment';
+
 export default Ember.Controller.extend(OSFAgnosticAuthControllerMixin, {
     toast: Ember.inject.service(),
+
+    signupUrl: `${config.OSF.url}login?campaign=${config.PREPRINTS.campaign}`,
```

Comment:
How quickly sleep deprived brains turn to mush... thanks for the reminder, and change pushed.